### DISCUSSION TITLE
Sphere tesselation

### DISCUSSION THREAD

**Discussion by LorenzE** [dipy/dipy#2403] [Preprocessing]
Hi all, 

I have a question regarding the pre-computed sphere tessellations. How were they actually created? Did you use subdivision of a Icosahedron maybe?

```
evenly_distributed_sphere_362.npz
evenly_distributed_sphere_642.npz
evenly_distributed_sphere_724.npz
repulsion724.npz
repulsion100.npz
repulsion200.npz
```

**Comment by skoudoro** [comment]
Hi @LorenzE, 

This is a good question for the historical maintainer!  @Garyfallidis and @arokem, Could you answer this question? Thank you
